sdk/python/tests/utils/online_read_write_test.py

```diff
@@ -89,18 +90,20 @@ def _driver_rw_test(event_ts, created_ts, write, expect_read):
 
     # Note: This behavior has changed for performance. We should test that older
     # value can't overwrite over a newer value once we add the respective flag
+    # TODO: add the `created_ts` tie breaker check in the Online stores
+    # TODO: switch order of _driver_rw_test to check that it's tie breaker logic vs. order of write
     """ created_ts is used as a tie breaker, using older created_ts here, but we still overwrite """
-    _driver_rw_test(
-        event_ts=time_1 + timedelta(hours=1),
-        created_ts=time_3 - timedelta(hours=1),
-        write=(54321, "I HAVE AN OLDER created_ts SO I LOSE"),
-        expect_read=(54321, "I HAVE AN OLDER created_ts SO I LOSE"),
-    )
+    # _driver_rw_test(
+    #     event_ts=time_1 + timedelta(hours=1),
+    #     created_ts=time_3 - timedelta(hours=1),
+    #     write=(54321, "I HAVE AN OLDER created_ts SO I LOSE"),
+    #     expect_read=(54321, "I HAVE AN OLDER created_ts SO I LOSE"),
+    # )
 
     """ created_ts is used as a tie breaker, using newer created_ts here so we should overwrite """
-    _driver_rw_test(
-        event_ts=time_1 + timedelta(hours=1),
-        created_ts=time_3 + timedelta(hours=1),
-        write=(96864, "I HAVE A NEWER created_ts SO I WIN"),
-        expect_read=(96864, "I HAVE A NEWER created_ts SO I WIN"),
-    )
+    # _driver_rw_test(
+    #     event_ts=time_1 + timedelta(hours=1),
+    #     created_ts=time_3 + timedelta(hours=1),
+    #     write=(96864, "I HAVE A NEWER created_ts SO I WIN"),
+    #     expect_read=(96864, "I HAVE A NEWER created_ts SO I WIN"),
+    # )
```

Comment:
I don't believe `created_ts` tie breaker was a feature in the recent feast versions
I think the only reason it passed was because it works in sequential write order not because of tie breaker logic